lighthouserc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    ci: {
+        collect: {
+            url: ['http://localhost:9090'],
```

Comment:
Wouldn't cube view be better testing url? Something like `http://localhost:9090/#covid19/`?

Reply:
Why not both?

Reply:
Why not, indeed!

Reply:
Because urls differ only by hash it kinda sucks and it cannot be collected in a single run. To put it simply, report for `/#covid19` will override report for `/`. For now I would stick with single url to `/covid19` for the sake of simplicity.

Reply:
Works for me. Cube view is more complex and definitely more often visited by users. A lot of the times they visit it straight from bookmarks.